docs/data/material/pages.ts

```diff
@@ -226,10 +226,6 @@ const pages: MuiPage[] = [
             pathname: '/material-ui/experimental-api/css-theme-variables/migration',
             title: 'Migrating to CSS variables',
           },
-          {
-            pathname: '/material-ui/experimental-api/css-variables',
```

Comment:
Great! We should also remove the `css-variables.md` file.

Reply:
Done